### ISSUE TITLE
Реквестирую добавление класса mp4/webm-file для mp4 файлов на дваче.

### ISSUE THREAD

**Issue by Megum1n** [SthephanShinkufag/Dollchan-Extension-Tools#1238]
Сейчас на дваче у mp4 файлов стоит хтмл класс **"img preview"**, в то время как у webm файлов **"img preview webm-file"**.
Из-за этого обводки у mp4 файлов нет и визуально отличить их от обычной картинки можно только по названию файла.
![image](https://user-images.githubusercontent.com/17014026/40479796-3bf584a6-5f55-11e8-8a10-308a47aa30b3.png)
Я не про в ксс/хтмл, пните если можно просто реализовать самому.

**Comment by SthephanShinkufag** [comment]
А я то думаю, чому у меня всё нормально.
Попробуйте скачать последний билд скрипта, я ковырял построение аякснутых постов для макабы, эта проблема уже исправлена.

https://dscript.me/

**Comment by SthephanShinkufag** [comment]
Если у вас хроморасширение, то я его попозже обновлю.

**Comment by Megum1n** [comment]
@SthephanShinkufag спасибо, няш, всё работает.

**Comment by Megum1n** [comment]
Судя по всему это Абу пофиксил. **class="img preview webm-file "** (с пробелом в конце) добавлен к мп4 файлам, так что и без куклы работает.

**Comment by SthephanShinkufag** [comment]
Нет. В аякснутых постах старый куклоскрипт не добавлял `webm-file`, так как раньше этого класса не было.
В недавнем коммите я этот класс добавил, егда подгонял пот тестовый движок макабы:
https://github.com/SthephanShinkufag/Dollchan-Extension-Tools/commit/c1c32bc43b27ff7efa0af3300e2cde5cf9f7c2c7#diff-89e61f0e70853f7cf8da992800a857ecR369

**Comment by SthephanShinkufag** [comment]
Проблема возникала только в аякснутых постах (загруженных по автообновлению треда, например) в старом куклоскрипте.
При чём тут работы без куклы.

**Comment by trollsasha1** [comment]
У меня до сих пор так с Mp4, версия последняя. Браузер хром.

**Comment by SthephanShinkufag** [comment]
@trollsasha1 

Исправил.
